cmd/agent/dist/conf.d/runtime_security_agent.d/default.policy

```diff
@@ -0,0 +1,53 @@
+- rule:
```

Comment:
my 2 cents, 
I think it can be good to version this file by adding a kind of header to the file structure. because if never this file can be provided by the user, it will ease to support backward and forrward compatibility between the agent and the file format.

```yaml
version: 1.0.0
rules:
  - id: credential_modified
    description: credential files modified using unknown tool
    expression: >-
      (open.filename == "/etc/shadow" || open.filename == "/etc/gshadow") &&
      process.name not in ["vipw", "vigr"]
    tags:
      mitre: T1003
```

Reply:
Good point. Updated

Reply:
Is there code to check the policy version is compatible with the agent?

Reply:
Not yet, we'll add this check when required